### ISSUE TITLE
Using variable GAME_CLIENT_PORT leaves the SERVER_LIST_PORT unexposed

### ISSUE THREAD

**Issue by Lunaresk** [Hermsi1337/docker-ark-server#14]
Whenever I used the variable GAME_CLIENT_PORT with a docker-compose.yml I noticed two things.
1. The port is not used in the starting arguments (there is no "\?Port=7001" when I want to use port 7001).
2. When I use this option, I can't find my server anymore via steam server interface, even tho I haven't changed this port.
Attached the malicious compose-file for possible reproduction.

I fixed it myself with entering the container and inserting the port option in the main.cfg from the instance.

[docker-compose.txt](https://github.com/Hermsi1337/docker-ark-server/files/7002072/docker-compose.txt)

**Comment by Hermsi1337** [comment]
Sorry for my very delayed response... 

It's very strange that as soon as you use the variable, you're unable to locate your server via steam.  
The variable is always set directly in the `Dockerfile` there should not be any difference if you also set it in `docker-compose.yml` without changing the value.

I didn't know, that it's required to run the server with the `port`-args when changing them.. that's why the arguments are missing.  
I'll provide a fix for that by always adding `?Port=${GAME_CLIENT_PORT}?QueryPort=${SERVER_LIST_PORT}` to the `arkmanager run`.

**Comment by Lunaresk** [comment]
Honestly I didn't know that either ^^' I just experimented a bit with some options and that's what I found. No problem with the delay.

**Comment by amelios** [comment]
Any chance this ever got updated? I was attempting to run 2 different servers, and the second one (with adjusted port numbers) seems to be colliding with the first. Guessing it's due to this.

**Comment by Hermsi1337** [comment]
As soon as I find the time, I will update this... 

Even though the change is not very extensive, I'm currently pretty busy..

But: You guys could try working this "bug" around by using additional args like

```shell
$ docker run hermsi/ark-server --arkopt,QueryPort=1234 --arkopt,Port=12345
```

I'd appreciate your Feedback if this works for you (=

**Comment by ThePowershellNinja** [comment]
@Hermsi1337 This issue seems to be solved in #25 with the symlink for the `/etc/arkmanager` directory to `/app/arkmanager`.

**Comment by Lunaresk** [comment]
It worked!! Awesome work :D
